rpc/client.go

```diff
@@ -446,10 +377,9 @@ func (c *Client) CallContextIgnoringLocalHandlers(ctx context.Context, result in
 
 	if c.router.routeRemote(method) {
 		client, err := c.getClientUsingCache(chainID)
-		if err != nil {
-			return err
+		if err == nil {
```

Comment:
why aren't we returning on `err != nil` now?

Reply:
Thank you! It's a dangerous change. I've reverted this and fixed the `transactor_test.go` by manually setting the client for the upstrea chainID via `SetClient`.